documentation/md/collection/layoutPositions.md

```diff
@@ -12,6 +12,7 @@ var options = {
   eles: someCollection, // collection of elements involved in the layout; set by cy.layout() or eles.layout()
   fit: true, // whether to fit the viewport to the graph
   padding: 30, // padding to leave between graph and viewport
+  nodeDimensionsIncludeLabels: false, // Boolean which changes whether label dimensions are included when calculating node dimensions
```

Comment:
We need to document the `node.layoutDimensions()` function.  While technically the option could also be read in `layoutPositions()`, the `layoutPositions()` function doesn't use it.